### PR TITLE
feat(site): initial scaffolding [WIP]

### PR DESCRIPTION
**Pull request by nothingismagick**
Just a start, needs a bit of text and TLC

### CODE REVIEW

**Comment by Beanow** [comment]
Please be sure to commit / generate a file `/docs/CNAME` containing `tauri-apps.org`.
As per [this doc](https://help.github.com/en/articles/configuring-a-publishing-source-for-github-pages#publishing-your-github-pages-site-from-a-docs-folder-on-your-master-branch) it should be in the folder not root of the master branch.

File: docs-generator/src/404.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<head>
+  <meta charset="utf-8" />
+
+  <title>404 Redirect</title>
+
+  <script>
+    sessionStorage.redirect = location.href;
+  </script>
+
+  <meta http-equiv="refresh" content="0;URL='/tauri'"></meta>
```
Comment (Beanow) [inline docs-generator/src/404.html]: Looks like this wants to go to `/tauri` but should probably go to the root `/` instead.

Comment (nothingismagick) [reply docs-generator/src/404.html]: ahhh - missed that one.
nice catch

**Comment by Beanow** [comment]
I'll leave the CNAME as an issue, since the DNS isn't set up for it right now anway.

**Review by Beanow — APPROVED** [review]

**Pull request merged**